src/py42/services/util.py

```diff
@@ -3,9 +3,9 @@
 
```

Comment:
In this file, the `convert_datetime_to_epoch()` method can have its implementation simplified to:

```
def convert_datetime_to_epoch(date):
    return date.timestamp()
```

The method is really not needed anymore, but we have to keep it around until py42 2.0. We plan to get rid of thi whole module then: https://github.com/code42/py42/projects/2#card-42594594